Makefile

```diff
@@ -1,60 +1,68 @@
-# You can follow the steps below in order to get yourself a local ODC.
-# Once running, you can access a Jupyter environment 
-# at 'http://localhost' with password 'secretpassword'
+## You can follow the steps below in order to get yourself a local ODC.
+## Start by running `setup` then you should have a system that is fully configured
+##
+## Once running, you can access a Jupyter environment
+## at 'http://localhost' with password 'secretpassword'
 
-# 1. Start your Docker environment
-up:
-	docker-compose up
+BBOX := 25,20,35,30
 
-# 2. Prepare the database
-init:
+help: ## Print this help
+	@grep -E '^##.*$$' $(MAKEFILE_LIST) | cut -c'4-'
+	@echo
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-10s\033[0m %s\n", $$1, $$2}'
+
+setup: build up init product index ## Run a full setup, or manually run the following 4 steps
+
+up: ## 1. Bring up your Docker environment
+	docker-compose up --detach postgres
+	docker-compose run checkdb
+	docker-compose up --detach jupyter
+
+init: ## 2. Prepare the database
 	docker-compose exec jupyter datacube -v system init
 
-# 3. Add a product definition for Sentinel-2
-product:
+product: ## 3. Add a product definition for Sentinel-2
 	docker-compose exec jupyter \
 		datacube product add \
 			https://raw.githubusercontent.com/digitalearthafrica/config/master/products/esa_s2_l2a.odc-product.yaml \
 			https://raw.githubusercontent.com/opendatacube/datacube-dataset-config/master/products/io_lulc.odc-product.yaml \
 			https://raw.githubusercontent.com/opendatacube/datacube-dataset-config/master/products/nasadem.odc-product.yaml
```

Comment:
Actually, we should change this to use `dc-sync-products`, which is a not yet documented tool.

HOpefully the test demonstrates how it can be used: https://github.com/opendatacube/odc-tools/blob/develop/apps/dc_tools/tests/test_add_update_products.py

Reply:
Let's just add a local one.